client/src/components/OnboardingParent/OnboardingParent.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "./OnboardingParent.css";
+import Logo from "../Logo/Logo.js";
+import CarouselDots from "../CarouselDots/CarouselDots";
+// Class Based React Component
```

Comment:
This is no longer a class based component - will you update this comment?

Reply:
Done